api4/post.go

```diff
@@ -983,13 +983,18 @@ func acknowledgePost(c *Context, w http.ResponseWriter, r *http.Request) {
 }
 
 func unacknowledgePost(c *Context, w http.ResponseWriter, r *http.Request) {
-	c.RequirePostId()
+	// license check
+	permissionErr := minimumProfessionalLicense(c)
+	if permissionErr != nil {
+		c.Err = permissionErr
+		return
+	}
+	c.RequirePostId().RequireUserId()
 	if c.Err != nil {
 		return
 	}
-	userId := r.URL.Query().Get("user_id")
 
-	if !c.App.SessionHasPermissionToUser(*c.AppContext.Session(), userId) {
+	if !c.App.SessionHasPermissionToUser(*c.AppContext.Session(), c.Params.UserId) {
 		c.SetPermissionError(model.PermissionEditOtherUsers)
 		return
 	}
```

Comment:
0/5, but I'd love to be able to use this API without having to always specify myself as a user. Just to confirm that we don't want to make the `user_id` an optional querystring parameter?

(I often would type things like `http POST :8065/api/v4/posts/<post_id/ack` at the CLI.)

Reply:
if we change the endpoint to include user_id in the url and be able to use `/user_id` or `/me`, would that work?